flexget/plugins/cli/series.py

```diff
@@ -34,69 +35,66 @@ def display_summary(options):
     :param options: argparse options from the CLI
     """
     session = Session()
```

Comment:
Good tip, will do. Is using the with_session wrapper preferable? does it matter?